lib/SILOptimizer/Mandatory/Differentiation.cpp

```diff
@@ -1444,8 +1444,18 @@ class DifferentiableActivityInfo {
   void setUseful(SILValue value, unsigned dependentVariableIndex);
```

Comment:
Note: currently, `setUseful` has two users (`setUsefulAndPropagateToOperands` and `propagateUsefulThroughAddress`), so it hasn't been inlined.